### ISSUE TITLE
Update Management on Docker Desktop

### ISSUE THREAD

**Issue by stephanierifai** [docker/roadmap#263]
**Tell us about your request**
Have more controls for updating Docker Desktop:
1. Be able to schedule an update for later when I am not in the middle of working
2. Enable auto updates of patch releases
3. Schedule updates on start of Desktop

**Which service(s) is this request for?**
Docker Desktop

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
I always get reminded about updating when I'm in the middle of trying to work, and I get frequent reminders, I want to be able to schedule it for a convenient time. 

**Are you currently working around the issue?**
Snoozing the reminder until I have time to update. 

**Additional context**
Are there other changes you want to updates? Let us know!

**Comment by vinnycrm** [comment]
Should have one tab for handling docker network

**Comment by vitor-l** [comment]
Improve Docker for Mac performance for M1 chip architectures... please! I'm begging you. I have MacBook M1 with 8 GB RAM and Docker for Mac is too slow. I used a notebook with lower settings, but with Linux, and it worked smoothly.

**Comment by stephen-turner** [comment]
@vitor-l Are you running amd64 (Intel architecture) containers on your M1? That will always be slow because they can't run natively and have to be emulated.

**Comment by delcroip** [comment]
on of those would be good 
Being able to export volumes based on a schedule
Being able to mark volume as "not eligible for prune"
Prune should not remove image/volume if there is a docker files using them

My biggest issue with docker is the disk management, after trying some software I and up with a lots of images and volumes, to reclaim the space there is the "prune" function but it also removes images/volumes of stopped containers. that is an issue because I also have application that I only run from time to time for which I don't want to lose volumes/images.

**Comment by fr3d3rico** [comment]
Sometimes, for example, when we start a new container like:
```
docker run -d -p 80:80 docker/getting-started
```
The container/apps tab doesnt show complete name. We have to click each one to see it because of few caracters:
![image](https://user-images.githubusercontent.com/1296366/134813044-18021475-7cbc-461b-8a2f-5d86cd5b3f4e.png)

* My case: Windows 10 , docker-desktop 4.0.1 and docker engine 20.10.8

Another issue that is simple to help beginners, like me, when we click to inspect container, it doesnt show more info, for example, network type and name.
Maybe it is an premium feature, but it is unclear.

**Comment by vitor-l** [comment]
@stephen-turner yes, I'm running amd64 containers. But almost all containers are amd64. So something needs to be done, and honestly, I find it easier to improve Docker's performance than to change multiple images.

**Comment by delfuego** [comment]
@vitor-l That's... just not how it works. You're running containers for the Intel processor on a non-Intel processor, so by definition, the need to be emulated. That emulation has zero to do with Docker; it's emulation provided by macOS. And it will never go away, because again, you're running containers on processors that they aren't compiled or designed for.

**Comment by mbayraktr** [comment]
I use Mac Air with M1 chip. I'm running amd64 containers and on backend server app crashes sometimes like that:
`error Command failed with exit code 137.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[nodemon] app crashed - waiting for file changes before starting...`

I increased the resources that container used but it did not fix the issue. Do you have an idea for this case ?
![Screen Shot 2021-09-28 at 13 13 04](https://user-images.githubusercontent.com/39880954/135068250-b63a5ee0-6ae2-4351-bd62-4a6e23be05d4.png)

**Comment by delfuego** [comment]
That's the node app that you're running in the container crashing; you'd have to debug what is happening in or to that app. At first glance, it looks to be an out-of-memory error… remember, you can give Docker more resources (e.g., memory), but you still have to then tell the container how much memory it should/can use.

**Comment by stephen-turner** [comment]
@mbayraktr I believe you've hit a known bug in previous versions of libssl. See https://docs.docker.com/desktop/mac/troubleshoot/#known-issues for more details.

**Comment by delfuego** [comment]
@stephen-turner No, that wouldn't be the case; that bug is on native (arm64) containers, not amd64 containers. And it shows up as a segfault, not an out-of-memory error (which is what that exit code 137 is).

**Comment by stephen-turner** [comment]
@delfuego Thanks for the correction. You're right, the libssl bug is only on arm64 containers. So it may just be another emulation problem.

**Comment by d3vel0per** [comment]
Should have a manual refresh button for containers tab and others if applicable. I have noticed many a times the containers dont appear in UI but viewable from cli or you just restart the UI and they appear again.

**Comment by scanaaron** [comment]
Support for M1 GPU and Neural Engines would be very helpful.
